app/src/main/java/org/wikipedia/richtext/CustomHtmlParser.kt

```diff
@@ -111,7 +111,6 @@ class CustomHtmlParser(private val handler: TagHandler) : TagHandler, ContentHan
         private var lastAClass = ""
         private var listItemCounts = Stack<Int>()
         private val listParents = mutableListOf<String>()
-        private val leadingMarginSize = DimenUtil.dpToPx(16f).toInt()
```

Comment:
This was breaking Previews in compose. Let's move this inline to where it's actually used.